projects/clr/hipamd/src/hip_log.cpp

```diff
@@ -0,0 +1,39 @@
+#include <hip/hip_runtime.h>
+#include "hip_internal.hpp"
+#include "hip_log.hpp"
+
+namespace hip {
+
+// Static instance pointer for LoggingInfo singleton
+LoggingInfo* LoggingInfo::lginfo_;
```

Comment:
Can this be part of the hip global singleton?

Reply:
Sure, I can check that.